code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/constants/ServiceDiscoveryConstants.java

```diff
@@ -68,6 +68,7 @@
     public static final String LABEL_SERVICE_GLOBAL = "io.rancher.scheduler.global";
     public static final String LABEL_SERVICE_REQUESTED_HOST_ID = "io.rancher.service.requested.host.id";
     public static final String LABEL_SERVICE_LAUNCH_CONFIG = "io.rancher.service.launch.config";
+    public static final String LABEL_SERVICE_CONTAINER_CREATE_ONLY = "io.rancher.service.container.start_once";
```

Comment:
Can you change to `io.rancher.container.start_once` to be more consistent with `io.rancher.container.hostname_override`